### PR TITLE
Update V8 to 8.4 and project to Visual Studio 2019

### PR DESCRIPTION
**Pull request by spahnke**

### CODE REVIEW

File: Source/Noesis.Javascript/JavascriptInterop.cpp [hunk 1]
```diff
@@ -537,15 +541,16 @@ JavascriptInterop::ConvertFromSystemDictionary(System::Object^ iObject)
 v8::Local<v8::Value>
 JavascriptInterop::ConvertFromSystemList(System::Object^ iObject) 
 {
-	v8::Isolate *isolate = JavascriptContext::GetCurrentIsolate();
+	auto isolate = JavascriptContext::GetCurrentIsolate();
+	auto context = isolate->GetCurrentContext();
```
Comment (Ablu) [inline Source/Noesis.Javascript/JavascriptInterop.cpp:545]: Is there any doc about this context now being required everywhere?

Comment (spahnke) [reply Source/Noesis.Javascript/JavascriptInterop.cpp:545]: There is never a doc about these breaking changes unfortunately. You basically can only look at the errors and the new signatures... If you find one, let me know 😃

**Comment by spahnke** [comment]
@oliverbock Can this be merged or is the update to VS 2019 a blocker for you?

**Comment by oliverbock** [comment]
Thanks for reminding  me.  Merged.

**Pull request merged**